site/content/docs/integrations/prometheus-v2.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Prometheus V2
+weight: 65
+---
+
+If you are using [Prometheus](https://prometheus.io/) for monitoring and the popular [Grafana](https://grafana.com/) stack
+for dashboarding, you can expose Checkly's core metrics on a dedicated, secured endpoint. 
+
+{{< info >}}
+This page describes a new V2 version of the Prometheus exporter. For information about the old Prometheus exporter, see the [Prometheus V1 docs](/docs/integrations/prometheus-v1/).
+{{</ info >}}
+
+## Activation
```

Comment:
I just copied this section from the V1 docs for now. I think that we should update the screenshots. Maybe there are other improvements possible here too.